Framework/Algorithms/inc/MantidAlgorithms/SofTwoThetaTOF.h

```diff
@@ -0,0 +1,43 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_ALGORITHMS_SOFTWOTHETATOF_H_
+#define MANTID_ALGORITHMS_SOFTWOTHETATOF_H_
+
+#include "MantidAPI/Algorithm.h"
+#include "MantidAlgorithms/DllConfig.h"
+
+namespace Mantid {
+namespace Algorithms {
+
+/** SofTwoThetaTOF : Convert a S(spectrum number, TOF) workspace to
+ * S(twoTheta, TOF) workspace.
+ */
+class MANTID_ALGORITHMS_DLL SofTwoThetaTOF : public API::Algorithm {
```

Comment:
I wonder if this would be better off as an `API::DataProcessorAlgorithm` as it is just automating the execution of several child algorithms while doing no direct data treatment itself.

Reply:
Makes sense.